.gitignore

```diff
@@ -2,5 +2,7 @@
 .sass-cache/
 bower_components/
 node_modules/
+demos/
```

Comment:
I see, sorry, I was told not to commit the files in root, I've fixed this
now.

On 25 March 2015 at 12:15, Alberto Elias <notifications@github.com> wrote:

> In .gitignore
> <https://github.com/Financial-Times/o-viewport/pull/17#discussion_r27114961>
> :
>
> > @@ -2,5 +2,7 @@
> >  .sass-cache/
> >  bower_components/
> >  node_modules/
> > +demos/
>
> You're supposed to get everything that's not in local
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Financial-Times/o-viewport/pull/17/files#r27114961>.
>

-- 

------------------------------
This email was sent by a company owned by Pearson plc, registered office at 
80 Strand, London WC2R 0RL.  Registered in England and Wales with company 
number 53723.